src/lsqecc/pipeline/lattice_surgery_compilation_pipeline.py

```diff
@@ -42,33 +42,58 @@ def compile_file(
 def compile_str(
     qasm_circuit: str, apply_litinski_transform: bool = True
 ) -> Tuple[List[GUISlice], str]:
-    """Returns gui slices and the text of the circuit as processed in various stages"""
+    """Returns gui slices and string JSON of compilation text)"""
     composer_class = lscc.LatticeSurgeryComputation
     layout_types = lscc.LayoutType
 
-    input_circuit = segmented_qasm_parser.parse_str(qasm_circuit)
-
-    compilation_text = "Input Circuit:\n"
-
-    compilation_text += qkvis.circuit_drawer(
+    input_circuit = qkvis.circuit_drawer(
```

Comment:
maybe worth calling `ascii_circuit`

Reply:
I am not familiar at all with qiskit, so I am open to all suggestions.

My angle of attack on this was to fix it in an agnostic way, but I think it's good to have verbose and explicit variable names at every step with comments just for later reads of the code.